### PR TITLE
[CHORE]Persistent login

### PR DESCRIPTION
**Pull request by kermit-klein**
[PT LINK](https://www.pivotaltracker.com/story/show/173205874)

- Add validateToken on page reload to keep user logged in on refresh

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -9,14 +9,27 @@ import LoginForm from "./components/LoginForm";
...
 import { getPlace } from "./modules/location";
+import auth from "./modules/auth";
 
 const App = (props) => {
   const [uid, setUid] = useState("");
   const [authenticated, setAuthenticated] = useState(false);
   const dispatch = useDispatch();
 
-  useEffect(() => {
+  useEffect(async () => {
```
Comment (ViaMarcus) [inline src/App.jsx:19]: Don't use async useEffect, factor out the login function to a separate async function, and call on it from within useEffect

**Review by ViaMarcus — CHANGES_REQUESTED** [review]

**Review by designerofthing — APPROVED** [review]

File: src/App.jsx (hunk 1 above)
Comment (erikbjoern) [reply src/App.jsx:19]: ^ I agree with this

Comment (erikbjoern) [reply src/App.jsx:19]: did you forget to remove it from here?

Comment (kermit-klein) [reply src/App.jsx:19]: No did not forget it wouldnt work, but removed now

**Review by jysmys — APPROVED** [review]

**Pull request merged**